lib/engine/game/g_18_norway/game.rb

```diff
@@ -312,10 +314,13 @@ def ship?(train)
           train.track_type == :narrow
         end
 
-        def cheapest_train_price
+        def cheapest_train
           depot_trains = depot.depot_trains.reject { |train| ship?(train) }
-          train = depot_trains.min_by(&:price)
-          train.price
+          depot_trains.min_by(&:price)
+        end
+
+        def cheapest_train_price
```

Comment:
This gets called with a parameter from the buy train step. It's not used here, but should be included in the method signature.
```suggestion
        def cheapest_train_price(_corporation)
```